lingpy/data/model.py

```diff
@@ -19,7 +19,7 @@ class Model(object):
     Parameters
```

Comment:
It's nice that you also work on documentation. A mental note for me here is that I should add "jaeger" to this, as this model is also readily available.